homeassistant/components/accuweather/const.py

```diff
@@ -61,145 +63,145 @@
-        ATTR_UNIT_IMPERIAL: PERCENTAGE,
+        "device_class": None,
+        "icon": "mdi:weather-lightning",
+        "label": "Thunderstorm Probability Night",
+        "unit_metric": PERCENTAGE,
+        "unit_imperial": PERCENTAGE,
     },
     "Tree": {
-        ATTR_DEVICE_CLASS: None,
-        ATTR_ICON: "mdi:tree-outline",
-        ATTR_LABEL: "Tree Pollen",
-        ATTR_UNIT_METRIC: CONCENTRATION_PARTS_PER_CUBIC_METER,
-        ATTR_UNIT_IMPERIAL: CONCENTRATION_PARTS_PER_CUBIC_METER,
+        "device_class": None,
+        "icon": "mdi:tree-outline",
+        "label": "Tree Pollen",
+        "unit_metric": CONCENTRATION_PARTS_PER_CUBIC_METER,
+        "unit_imperial": CONCENTRATION_PARTS_PER_CUBIC_METER,
     },
     "UVIndex": {
-        ATTR_DEVICE_CLASS: None,
-        ATTR_ICON: "mdi:weather-sunny",
-        ATTR_LABEL: "UV Index",
-        ATTR_UNIT_METRIC: UV_INDEX,
-        ATTR_UNIT_IMPERIAL: UV_INDEX,
+        "device_class": None,
+        "icon": "mdi:weather-sunny",
+        "label": "UV Index",
+        "unit_metric": UV_INDEX,
+        "unit_imperial": UV_INDEX,
     },
     "WindGustDay": {
-        ATTR_DEVICE_CLASS: None,
-        ATTR_ICON: "mdi:weather-windy",
-        ATTR_LABEL: "Wind Gust Day",
-        ATTR_UNIT_METRIC: SPEED_KILOMETERS_PER_HOUR,
-        ATTR_UNIT_IMPERIAL: SPEED_MILES_PER_HOUR,
+        "device_class": None,
+        "icon": "mdi:weather-windy",
+        "label": "Wind Gust Day",
+        "unit_metric": SPEED_KILOMETERS_PER_HOUR,
+        "unit_imperial": SPEED_MILES_PER_HOUR,
     },
     "WindGustNight": {
-        ATTR_DEVICE_CLASS: None,
-        ATTR_ICON: "mdi:weather-windy",
-        ATTR_LABEL: "Wind Gust Night",
-        ATTR_UNIT_METRIC: SPEED_KILOMETERS_PER_HOUR,
-        ATTR_UNIT_IMPERIAL: SPEED_MILES_PER_HOUR,
+        "device_class": None,
+        "icon": "mdi:weather-windy",
+        "label": "Wind Gust Night",
+        "unit_metric": SPEED_KILOMETERS_PER_HOUR,
+        "unit_imperial": SPEED_MILES_PER_HOUR,
     },
     "WindDay": {
-        ATTR_DEVICE_CLASS: None,
-        ATTR_ICON: "mdi:weather-windy",
-        ATTR_LABEL: "Wind Day",
-        ATTR_UNIT_METRIC: SPEED_KILOMETERS_PER_HOUR,
-        ATTR_UNIT_IMPERIAL: SPEED_MILES_PER_HOUR,
+        "device_class": None,
+        "icon": "mdi:weather-windy",
+        "label": "Wind Day",
+        "unit_metric": SPEED_KILOMETERS_PER_HOUR,
+        "unit_imperial": SPEED_MILES_PER_HOUR,
     },
     "WindNight": {
-        ATTR_DEVICE_CLASS: None,
-        ATTR_ICON: "mdi:weather-windy",
-        ATTR_LABEL: "Wind Night",
-        ATTR_UNIT_METRIC: SPEED_KILOMETERS_PER_HOUR,
-        ATTR_UNIT_IMPERIAL: SPEED_MILES_PER_HOUR,
+        "device_class": None,
+        "icon": "mdi:weather-windy",
+        "label": "Wind Night",
+        "unit_metric": SPEED_KILOMETERS_PER_HOUR,
+        "unit_imperial": SPEED_MILES_PER_HOUR,
     },
 }
 
-OPTIONAL_SENSORS = (
+OPTIONAL_SENSORS: Final[tuple[str, ...]] = (
```

Comment:
Why is this tuple and not list? Semantically looks like a list.

Reply:
Good question, I don't remember. I'll remove this tuple and add `enabled` key for type sensors dicts.